packages/drawer/src/views/Drawer.tsx

```diff
@@ -516,7 +516,7 @@ export default class DrawerView extends React.Component<Props> {
         oldState: (s: Animated.Value<number>) =>
           cond(
             eq(s, GestureState.ACTIVE),
-            set(this.manuallyTriggerSpring, TRUE)
+            call([], () => this.toggleDrawer(false))
```

Comment:
This will trigger callback to JS which can make the drawer lag if JS thread is busy. We need to figure out a different solution.

Reply:
A lag when tapping?  I'm not sure a lag when tapping would be noticeable (vs a lag when releasing a drag gesture).

Reply:
it might be noticeable on lower end devices, similar to how touchables don't feel very responsive